x-pack/plugins/apm/server/lib/helpers/setup_request.ts

```diff
@@ -40,10 +40,14 @@ interface APMRequestQuery {
 export function setupRequest(req: Legacy.Request): Setup {
   const query = (req.query as unknown) as APMRequestQuery;
   const cluster = req.server.plugins.elasticsearch.getCluster('data');
+  const uiSettings = req.getUiSettingsService();
+
+  const client: ESClient = async (type, params) => {
+    const includeFrozen = await uiSettings.get('search:includeFrozen');
 
-  const client: ESClient = (type, params) => {
     if (query._debug) {
       console.log(`DEBUG ES QUERY:`);
+      console.log({ includeFrozen });
```

Comment:
console.log

Reply:
Yes, that's on purpose as part of the debug output (otherwise we have no way to confirm if we're sending the right value to ES, AFAICT)

Reply:
Oh, this is on purpose for debugging?